pytorch_lightning/utilities/types.py

```diff
@@ -44,4 +44,5 @@
 ]
 EVAL_DATALOADERS = Union[DataLoader, Sequence[DataLoader]]
 LRSchedulerTypeTuple = (_LRScheduler, ReduceLROnPlateau)
+LRSchedulerTypeUnion = Union[_LRScheduler, ReduceLROnPlateau]
```

Comment:
for someone who's looked at this for the first time, it's pretty confusing having `LRSchedulerTypeTuple`, `LRSchedulerTypeUnion`, and `LRSchedulerType` . i couldn't tell the difference between these based on the names. i do prefer typing the LR schedulers in a separate PR

Reply:
maybe rename `LRSchedulerType` --> `LRSchedulerTypeClassesUnion` ? also agree this may be better in a separate PR

Reply:
could you add a TODO here? we can merge this and then improve the LR scheduler typing separately